starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -707,17 +712,18 @@ sub check_dither {
 
     $self->{dither_acq} = $acq_dither;
     $self->{dither_guide} = $guide_dither;
+    $self->{dither_guide_y_max} = $guide_dither->{ampl_y};
+    $self->{dither_guide_z_max} = $guide_dither->{ampl_p};
 
     # Check for standard dither
     if ($guide_dither->{state} eq 'ENAB'){
         if ((not standard_dither($guide_dither)) or not standard_dither($acq_dither)){
             push @{$self->{yellow_warn}}, "$alarm Non-standard dither\n";
         }
-        else{
-            if (($guide_dither->{ampl_p} != $acq_dither->{ampl_p})
-                    or ($guide_dither->{ampl_y} != $acq_dither->{ampl_y})){
-                push @{$self->{yellow_warn}}, "$alarm Dither changes between ACQ and GUI \n";
-            }
+        if (($guide_dither->{ampl_p} != $acq_dither->{ampl_p})
+                or ($guide_dither->{ampl_y} != $acq_dither->{ampl_y})){
+            push @{$self->{yellow_warn}}, sprintf("$alarm Reviewed with ACQ dither Y=%.1f Z=%.1f \n",
```

Comment:
The seems like an Info message to me.  There is nothing wrong or actionable here, just giving the reviewer a heads-up of something slightly unusual.

Reply:
There's the question on whether a real "heads-up" should be yellow.  I was mostly thinking that the action is that should get some extra attention and the question is if that extra attention will happen without with a yellow warning count in the summary.